README.md

```diff
@@ -1,21 +1,18 @@
 # conjur-api-python3
 
-Python3-based API SDK for [Conjur OSS](https://www.conjur.org/). The repo
```

Comment:
Conjur CLI / SDK lets you access and manage Conjur resources from either the command line or from your application

Reply:
Using the Conjur CLI / SD,  you can access and manage Conjur resources from the command line or from your application